packages/manager/src/features/CancelLanding/CancelLanding.tsx

```diff
@@ -37,7 +43,10 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+type CombinedProps = FeatureFlagConsumerProps;
+
 export const CancelLanding: React.FC<{}> = () => {
+  const flags = useFlags();
```

Comment:
```suggestion
const { flags } = props;
```

(made available by the flag wrapper/HOC -- let's just rely on that instead of using the `useFlags()` hook)